common/tests/test_middleware.py

```
@@ -0,0 +1,129 @@
+    return mw, get_response, response
+
+
+class RequestIDMiddlewarePropagationTests(TestCase):
+    def setUp(self):
+        self.factory = RequestFactory()
+
+    def test_existing_header_is_preserved(self):
+        """Client-supplied X-Request-ID must not be replaced."""
+        client_id = "my-client-request-id-123"
+        request = self.factory.get("/", HTTP_X_REQUEST_ID=client_id)
+        mw, get_response, response = _make_middleware()
```

Ruff (0.15.10)

[warning] Unpacked variable `get_response` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)